src/emoji.ts

```diff
@@ -15,14 +15,14 @@ limitations under the License.
 */
 
 import EMOJIBASE from 'emojibase-data/en/compact.json';
+import SHORTCODES from 'emojibase-data/en/shortcodes/iamcal.json';
```

Comment:
`compact.json` already contains shortcodes, ignoring them is not space-efficient. We should either use a file which doesn't already contain them or ideally as a build step make a customised JSON file which has exactly the data we need.

Reply:
That is no longer the case in Emojibase 6 - shortcodes were explicitly removed from compact.json with the intention that you'd be importing those separately

Reply:
https://github.com/milesj/emojibase/blob/master/packages/data/CHANGELOG.md

Reply:
Great!